funcions/1.js

```diff
@@ -0,0 +1,23 @@
+function oldRange(x, y) {
+	var oldArray = []
+	if (typeof x !== 'number' && typeof y !== 'number') {
+		throw new Error('Only numbers can be provided!')
+	} else {
+		for (var i = x; i < y; i++) {
+			oldArray.push(i)
+		}
+	}
+	return oldArray
+}
+
+const newRange = (start, stop, step = 1) =>
```

Comment:
Nitpick: you don't seem to be using/testing the `step` argument, is it really needed?